lib/expand/bootstrap.go

```diff
@@ -17,17 +17,33 @@ limitations under the License.
 package expand
 
 import (
+	"net/url"
+	"strings"
+
 	"github.com/gravitational/gravity/lib/install"
 	"github.com/gravitational/gravity/lib/ops"
+	rpcserver "github.com/gravitational/gravity/lib/rpc/server"
 	"github.com/gravitational/gravity/lib/state"
 	"github.com/gravitational/gravity/lib/storage"
 	"github.com/gravitational/gravity/lib/system/environ"
+	log "github.com/sirupsen/logrus"
 
 	"github.com/gravitational/trace"
 )
 
 // init initializes the peer after a successful connect
 func (p *Peer) init(ctx operationContext) error {
+	err := p.initEnviron(ctx)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	if !ctx.hasOperation() {
+		return nil
```

Comment:
this is a special case for expand when we don't have the operation after connecting to the cluster (CLI-workflow when the operation is created by the agent).